src/transforms/branch.ts

```diff
@@ -37,31 +32,33 @@ async function getLastCommit(github: GitHubAPI, webhookPayload: WebhookPayloadCr
 		url,
 		updateSequenceId: Date.now()
 	};
-}
+};
 
-export const transformBranch = async (github: GitHubAPI, webhookPayload: WebhookPayloadCreate) => {
-	if (webhookPayload.ref_type !== "branch") return undefined;
+export const transformBranch = async (github: GitHubAPI, webhookPayload: WebhookPayloadCreate, jiraHost: string, installationId: number, logger: LoggerWithTarget): Promise<JiraBranchData | undefined> => {
+	if (webhookPayload.ref_type !== "branch") {
+		return;
+	}
 
 	const { ref, repository } = webhookPayload;
-
 	const issueKeys = issueKeyParser().parse(ref) || [];
 
-	if (_.isEmpty(issueKeys)) {
-		return undefined;
+	if (isEmpty(issueKeys)) {
+		return;
 	}
 
-	const lastCommit = await getLastCommit(github, webhookPayload, issueKeys);
-
-	const newPrUrl = await booleanFlag(BooleanFlags.USE_NEW_GITHUB_PULL_REQUEST_URL_FORMAT, false);
+	const gitHubClient = new GitHubClient(getCloudInstallationId(installationId), logger);
+	const newPrUrl = booleanFlag(BooleanFlags.USE_NEW_GITHUB_PULL_REQUEST_URL_FORMAT, false, jiraHost);
+	const newGitHubClient = booleanFlag(BooleanFlags.USE_NEW_GITHUB_CLIENT_FOR_BRANCH_EVENT, false, jiraHost);
+	const [useNewPrUrl, useNewGitHubClient] = await Promise.all([newPrUrl, newGitHubClient]);
```

Comment:
/nit I've never been a fan of using constants that don't stick around.

Reply:
sounds reasonable, will convert